sdk/monitor/azure-monitor-opentelemetry-exporter/README.md

```diff
@@ -29,50 +29,79 @@ In order to export telemetry data to Azure Monitor, you will need the instrument
 search for your resource. On the overview page of your resource, you will find the instrumentation key in the top
 right corner.
 
-### Creating exporter builder for Azure Monitor
-```java readme-sample-createExporterBuilder
-AzureMonitorExporterBuilder azureMonitorExporterBuilder = new AzureMonitorExporterBuilder()
-    .connectionString("{connection-string}");
+
+### Setup the OpenTelemetry SDK to work with Azure Monitor exporter
+
+The following code shows how to configure the OpenTelemetry SDK auto-configuration with the Azure Monitor exporter:
+```java readme-sample-autoconfigure
+AutoConfiguredOpenTelemetrySdkBuilder sdkBuilder = AutoConfiguredOpenTelemetrySdk.builder();
+new AzureMonitor("{connection-string}")
+    .configure(sdkBuilder);
+OpenTelemetry openTelemetry = sdkBuilder.build().getOpenTelemetrySdk();
 ```
 
-#### Exporting span data
+## Examples
 
-The following example shows how to export a trace data to Azure Monitor through the
- `AzureMonitorTraceExporter`
+The following sections provide code samples using the OpenTelemetry Azure Monitor Exporter client library and OpenTelemetry API.
 
-##### Setup OpenTelemetry SDK to work with Azure Monitor exporter
-```java readme-sample-setupExporter
-// Create Azure Monitor exporter and initialize OpenTelemetry SDK
-// This should be done just once when application starts up
-AutoConfiguredOpenTelemetrySdkBuilder sdkBuilder = AutoConfiguredOpenTelemetrySdk.builder();
 
-new AzureMonitorExporterBuilder()
-    .connectionString("{connection-string}")
-    .install(sdkBuilder);
+The following example shows how create a span:
 
-OpenTelemetry openTelemetry = sdkBuilder.build().getOpenTelemetrySdk();
+```java 
```

Comment:
please add a code sample and then reference it as a snippet similar to the previous one

```
```java readme-sample-autoconfigure
```

then the code will be compiled and checked and the snipped will be auto-populated